test/integration/replay/postback_test.go

```diff
@@ -39,77 +31,83 @@ func TestMain(m *testing.M) {
 	code := m.Run()
 	cancel()
 	app.WaitForCleanup(ctx)
-	os.Exit(code)
+	_ = code
+	//os.Exit(code)
+	os.Exit(-1)
 }
 
 func doReplay(t *testing.T, f func(*builder.Builder)) error {
-	ctx := log.Testing(t)
-
-	r := bind.NewRegistry()
-	ctx = bind.PutRegistry(ctx, r)
-
-	ctx = database.Put(ctx, database.NewInMemory(ctx))
-
-	device := bind.Host(ctx)
-	client := gapir.New(ctx)
-	abi := device.Instance().GetConfiguration().PreferredABI(nil)
-	os := device.Instance().GetConfiguration().GetOS()
-	connection, err := client.Connect(ctx, device, abi)
-	if err != nil {
-		t.Errorf("Failed to connect to '%v': %v", device, err)
-		return err
-	}
-
-	b := builder.New(abi.MemoryLayout, nil)
-
-	f(b)
-
-	payload, decoder, notification, err := b.Build(ctx)
-	if err != nil {
-		t.Errorf("Build failed with error: %v", err)
-		return err
-	}
-
-	err = executor.Execute(ctx, payload, decoder, notification, connection, abi.MemoryLayout, os)
-	if err != nil {
-		t.Errorf("Executor failed with error: %v", err)
-		return err
-	}
-
+	// TODO(awoloszyn): Fix this replay test. The executor API
```

Comment:
Is this something you plan to fix soon? Otherwise, please create a ticket for the remaining tests.

Reply:
Absolutely, this was a first attempt at getting everything building. I have a bunch of changes coming up to fix all of the tests.